src/App/Pages/Accounts/LoginSsoPageViewModel.cs

```diff
@@ -0,0 +1,177 @@
+﻿using Bit.App.Abstractions;
+using Bit.App.Resources;
+using Bit.Core;
+using Bit.Core.Abstractions;
+using Bit.Core.Utilities;
+using System;
+using System.Threading.Tasks;
+using Bit.Core.Enums;
+using Bit.Core.Models.Domain;
+using Xamarin.Essentials;
+using Xamarin.Forms;
+
+namespace Bit.App.Pages
+{
+    public class LoginSsoPageViewModel : BaseViewModel
+    {
+        private const string Keys_RememberedOrgIdentifier = "rememberedOrgIdentifier";
+        private const string Keys_RememberOrgIdentifier = "rememberOrgIdentifier";
```

Comment:
I didn't make a checkbox for this on web. Maybe we should

Reply:
Do you want to present the option on mobile?  (there's currently no toggle for saving email on the login page so I followed that pattern)

Reply:
You show a toggle for rememebr org identifier, or no?

Reply:
Currently no.

Reply:
Ok, leave it as-is then.